moment_kinetics/src/fokker_planck.jl

```diff
@@ -68,6 +70,46 @@ using ..fokker_planck_test: Cssp_fully_expanded_form, calculate_collisional_flux
 using ..fokker_planck_test: d2Gdvpa2_Maxwellian, d2Gdvperpdvpa_Maxwellian, d2Gdvperp2_Maxwellian, dHdvpa_Maxwellian, dHdvperp_Maxwellian
 using ..fokker_planck_test: F_Maxwellian, dFdvpa_Maxwellian, dFdvperp_Maxwellian
 
+"""
+Function for reading Fokker Planck collision operator input parameters. 
+Structure the namelist as follows.
+
+[fokker_planck_collisions]
+use_fokker_planck = true
+nuii = 1.0
+frequency_option = "manual"
+"""
+function setup_fkpl_collisions_input(toml_input::Dict, reference_params)
+    # get reference collision frequency (note factor of 1/2 due to definition choices)
+    nuii_fkpl_default = 0.5*get_reference_collision_frequency(reference_params)
+    # read the input toml and specify a sensible default
+    input_section = set_defaults_and_check_section!(toml_input, "fokker_planck_collisions",
+       # begin default inputs (as kwargs)
+       use_fokker_planck = false,
+       nuii = -1.0,
+       frequency_option = "manual")

```

Comment:
I think the Krook collisions and FP collisions should have the same default setting for `frequency_option`, either both `"manual"` or both `"reference_parameters"`. At the moment Krook has `"reference_parameters"` and FP has `"manual"`.

Should we vote on which to choose? I'd vote for `"reference_parameters"` as the default, to make it easier to have self-consistent inputs. I guess @mrhardman would vote the other way!

Reply:
I would prefer manual input as default, but I can understand your reasoning. Let's change the default to `"reference_parameters"` on the basis that I will remember to ask for the `"manual"` option.